draft-wussler-openpgp-forwarding.md

```diff
@@ -332,10 +392,17 @@ Section 5.2.3.26.
 In case of collusion between the proxy and forwardee, an adversary may only be
 able to decrypt other messages, but not authenticate, sign, or certify other
 keys as the recipient.
+// This is redundant and partially contradictory with the section about
+// Generating the forwardee key, which says that the recipient may have
+// 0x10 as well. I also don't see why it can't have 0x08 (encrypt storage)
+// too. I would try to remove this section and merge it into the others,
+// if possible.
 
 The forwardee encryption subkey MUST be flagged with 0x40 and 0x10 only,
 this will prevent other implementations from sending messages directly to this
 key, causing decryption errors when using the wrong fingerprint in the KDF.
```

Comment:
Strictly speaking, these flags don't prevent key export, they just prevent implementations from using the key in the event of key export, so I would say that instead.
The run on clause "... causing decryption errors" also contradicts the first half of the sentence, as there can't be decryption errors if encryption is prevented :) So I would either drop this, or say more explicitly that even in the event that a message is encrypted to it directly, decryption would fail.

Reply:
There is in the text the following requirement
```
An implementation SHOULD NOT export public subkeys flagged as 0x40.
A public key directory SHOULD NOT accept subkeys flagged as 0x40.
```
thus why "prevent public key export"


Forwardee encryption subkeys have flags 0x40 and 0x10 only, in order to prevent
forwarding-capable implementation from exporting the public key and stop 
other implementations from encrypting messages directly to this key.